unit_tests/client/test_client.py

```diff
@@ -21,25 +21,33 @@ def child_chain(self):
 
     @pytest.fixture(scope='function')
     def client(self, root_chain, child_chain):
-        return Client(root_chain, child_chain)
+        DUMMY_KEY = 'key'
```

Comment:
Minor: I know I am the one that start the style of mocking everything. However, if by using some real key value and we can avoid those mocks on eg. `when('plasma_cash.client.client.utils').privtoaddr(client.key).thenReturn(DUMMY_XXX)`, it might be easier to read the test and the test also covers better. And the effort of using real key is not that hard as well (supposedly). However, this is just my opinion (recently got). Would leave this to you.